components/channel_selector_modal/channel_selector_modal.tsx

```diff
@@ -131,18 +137,22 @@ export default class ChannelSelectorModal extends React.PureComponent {
         }
     }
 
-    handleDelete = (values) => {
+    handleDelete = (values: ChannelWithTeamData[]) => {
```

Comment:
Should this be an array of `ChannelWithTeamData & Value` ?

Reply:
I've updated the PR with this suggestion.